src/View/View.php

```diff
@@ -1255,11 +1255,14 @@ protected function _paths($plugin = null, $cached = true)
             }
         }
 
+        $class = new \ReflectionClass('\Cake\View\View');
+        $cake = dirname(dirname($class->getFilename()));
```

Comment:
You guys are right, I got confused, sorry about that. Calling `parent::_paths()` works fine. I didn't realize but the source of my own issue was that I overrided `_path()` by copy/pasting the method in a child class, which ended up giving different results for obvious reasons.

Not sure how much of an issue it is at this point. It work as expected, though it might go unnoticed by someone overriding the method like I did.